tests/I24/serial/setup_beamline/test_zebra_plans.py

```diff
@@ -119,11 +122,29 @@ async def test_setup_zebra_for_fastchip(zebra: Zebra, RE):
     )
     # Check ttl out2 is set to AND3
     assert await zebra.output.out_pvs[2].get_value() == AND3
-    assert await zebra.pc.pulse_width.get_value() == exposure_time / 2
 
+    assert await zebra.pc.pulse_start.get_value() == 0.0
+    assert await zebra.pc.pulse_width.get_value() == exposure_time / 2
     assert await zebra.pc.pulse_step.get_value() == exposure_time + 0.0001
 
 
+async def test_open_fast_shutter_at_each_position_plan(zebra: Zebra, RE):
+    num_exposures = 2
+    exposure_time = 0.001
+
+    RE(open_fast_shutter_at_each_position_plan(zebra, num_exposures, exposure_time))
+
+    # Check output Pulse2 is set
+    assert await zebra.output.pulse_2.input.get_value() == PC_GATE
+    assert await zebra.output.pulse_2.delay.get_value() == 0.0
+    # num_exp * exposure_time + 0.05 (SHUTTER_OPEN_TIME)
+    assert await zebra.output.pulse_2.width.get_value() == pytest.approx(
+        0.052, abs=1e-3
+    )
```

Comment:
Could: I think it's cleaner to do the maths here properly i.e.
```suggestion
    expected_pulse_width = num_exp * exposure_time + 0.05 
    assert await zebra.output.pulse_2.width.get_value() == pytest.approx(
        expected_pulse_width, abs=1e-3
    )
```